EdFi.Ods.Utilities.Migration/Configuration/ConfigurationAutoMapper.cs

```diff
@@ -102,12 +102,13 @@ public class ConfigurationAutoMapperProfile : Profile
     {
         public ConfigurationAutoMapperProfile()
         {
-            CreateMap<MigrationConfigurationGlobal, MigrationConfigurationV311ToV34>();
-            CreateMap<MigrationConfigurationGlobal, MigrationConfigurationV33ToV34>();
-            CreateMap<MigrationConfigurationGlobal, MigrationConfigurationV311ToV33>();
-            CreateMap<MigrationConfigurationGlobal, MigrationConfigurationV31ToV311>();
-            CreateMap<MigrationConfigurationGlobal, MigrationConfigurationV25ToV31>();
             CreateMap<MigrationConfigurationGlobal, MigrationConfigurationV24ToV25>();
+            CreateMap<MigrationConfigurationGlobal, MigrationConfigurationV25ToV31>();
```

Comment:
This seems to imply that we didn't fully finish ODS-4128, the entire purpose of that ticket was to avoid the permutation explosion of classes for handling every variation of migration.

Reply:
It looks like we didn't implement the chained migration manager that would have supported this scenario. I'll open a ticket to address this.